app/src/main/java/com/scv/slackgo/helpers/ChannelListHelper.java

```diff
@@ -92,4 +96,28 @@ public boolean evaluate(Integer index) {
             return new ArrayList<String>();
         }
     }
+
+    public static List<String> channelsByIDFromTextViewString(List<String> channelsByName, List<Channel> channels) {
+        List<String> channelsById = new ArrayList<String>();
+        for (Channel channel : channels ){
```

Comment:
I thinks is more declarative, but maybe if we use just channels.... Is it possible?